content/blog/2019/11/2019-11-21-welcome-to-the-matrix.adoc

```diff
@@ -343,6 +343,7 @@ pipeline {
 ----
 
 
+If I run this Pipeline from the Jenkins UI and set the `PLATFORM_FILTER` parameter to `mac` get something like the output below:
```

Comment:
If I run this Pipeline from the Jenkins UI and set the `PLATFORM_FILTER` parameter to `mac`, I'll get something like the output below:

Reply:
@kshultzCB Is  the suggestion button not enabled for you?

Reply:
No, it's not. Every time I see one of those Suggestions done by someone, I wonder why I never get the option.